packages/fx-core/src/plugins/resource/aad/errors.ts

```diff
@@ -149,3 +149,28 @@ export class GraphClientErrorMessage {
   static readonly AppObjectIdIsNull = "Missing Object ID.";
   static readonly EmptyResponse = "Missing response.";
 }
+
+export const GraphErrorCodes = new Map([
+  [
```

Comment:
Could we store the content into a resource file? This might be easier to maintain. You can decide which way to use based on the pros/cons comparison.

Reply:
Fixed. Actually I thought it may be better to put errors in one file. For better maintenance, it should be better to put them in a file.

Reply:
I means put graph error codes into a xxx.json file, and then load it into memory in our code. I raise this comment because we seems need to add a lot of error codes here.

Reply:
Will update this when adding error code here.